src/main/java/de/teamlapen/vampirism/player/vampire/VampirePlayer.java

```diff
@@ -707,6 +755,14 @@ public void onUpdate() {
             } else {
                 ticksInSun = 0;
             }
+            if (player.ticksExisted % 5 == 0 && victim != -1) {
+                spawnBiteParticle(victim);
```

Comment:
This is quite annoying.
The server already triggers a sound when the blood is actually transfered, that should be enough I think.
Alternative would be to do use actual blood particles flying from the biten entity to the player simlar to what the altar of  inspiration does https://github.com/TeamLapen/Vampirism/blob/218a791e30962cd2c5e650b14fc34ceca954b1c7/src/main/java/de/teamlapen/vampirism/tileentity/TileAltarInspiration.java#L82-L82

Reply:
fixed as of latest commit.  I removed the four times a second bite particle and added the altar of inspiration blood particles each bite.  The end of the particles is delayed enough that its pretty constant in between bites.